client/components/domains/registrant-extra-info/docs/example.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+const countriesList = require( 'lib/countries-list' ).forDomainRegistrations(),
+	Card = require( 'components/card' ),
+	ExtraInfoFrForm = require( 'components/domains/registrant-extra-info/fr-form' ).default;
+
+class ExtraInfoFrFormExample extends PureComponent {
+	static displayName = 'DomainsFrExtraForm'
+
+	constructor( props ) {
+		super( props );
+
+		this.state = {
```

Comment:
FYI: no `static`, it's just:
```
	state = {
		whatever: 'value',
	}
```
Which does make sense (we want the value on the object not the prototype), but it's kind of weird syntax to be just running code there (I stuck a `console.log()` in there to check, and it is just running that code before the constructor with the instance as `this`).

Thinking about it though, this is the same thing as the binding.